playbooks/ci-full-rhel/tasks/create_heat_template.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: create heat template for test env
```

Comment:
this is not needed at all we don't use it anymore as we have heat templates in the envs. we should remove it to reduce potential for confusion.